runtimes/peregrine/src/weights/did.rs

```diff
@@ -723,6 +722,38 @@ impl<T: frame_system::Config> did::WeightInfo for WeightInfo<T> {
 			.saturating_add(T::DbWeight::get().reads(4))
 			.saturating_add(T::DbWeight::get().writes(3))
 	}
+
+	/// Storage: Did Did (r:1 w:0)
+	/// Proof: Did Did (max_values: None, max_size: Some(2312), added: 4787, mode: MaxEncodedLen)
+	fn dispatch_as() -> Weight {
+		// Proof Size summary in bytes:
+		//  Measured:  `353`
+		//  Estimated: `5777`
+		// Minimum execution time: 186_836_000 picoseconds.
+		Weight::from_parts(189_377_000, 0)
```

Comment:
weights will be calculated on our benchmark server. There were done with debug builds